docs/Systems/Tailwind/components/UtilityClassTemplate.tsx

```diff
@@ -42,7 +41,7 @@ export const UtilityClassTemplate = ({
               >
                 <div className="mr-32 min-w-max max-w-[300px]">
                   <CodeSnippet
-                    text={utilityClassName}
+                    text={utilityClassName.replace("-DEFAULT", "")}
```

Comment:
We don't want the `-DEFAULT` postfix on the utility class showing up in the code snippets on the docs.